Makefile

```diff
@@ -24,57 +24,43 @@ BUILD_VERSION ?=
 BUILD_TAG ?=
 BOARD_FILE ?=
 SCENARIO_FILE ?=
+CONFIG_XML_ENABLED ?= false
 
 export TOOLS_OUT
 
 .PHONY: all hypervisor devicemodel tools doc
 all: hypervisor devicemodel tools
 
 ifneq ($(BOARD_FILE)$(SCENARIO_FILE),)
-BOARD := `sed -n '/board/p' $(BOARD_FILE) |head -1|awk -F'"' '{print $$2}'`
-SCENARIO := `sed -n '/scenario/p' $(SCENARIO_FILE) |head -1|awk -F'"' '{print $$4}'`
+override BOARD := $(shell echo `sed -n '/<acrn-config/p' $(BOARD_FILE) | sed -r 's/.*board="(.*)".*/\1/g'`)
+override SCENARIO := $(shell echo `sed -n '/<acrn-config/p' $(SCENARIO_FILE) | sed -r 's/.*scenario="(.*)".*/\1/g'`)
 
-ifeq ($(BOARD), apl-nuc)
-override BOARD := nuc6cayh
-else ifeq ($(BOARD), kbl-nuc-i7)
-override BOARD := nuc7i7dnb
+ifneq ($(BOARD)$(SCENARIO),)
```

Comment:
What if the BOARD / SCENARIO is specified instead of BOARD_FILE / SCENARIO_FILE?